common/base64.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _BASE64_H
+#define _BASE64_H
+
+extern int __b64_ntop(u_char const *src, size_t srclength, char *target, size_t targsize);
```

Comment:
Instead of changing function names from the original `base64.c` file, would it be good for you if we change the `#define`:
```
#define rfbBase64NtoP __b64_ntop
#define rfbBase64PtoN __b64_pton
```